resources/styles/components/task/breadcrumbs.less

```diff
@@ -28,39 +31,62 @@
   }
 }
 
-.steps {
-  margin-left: -2px;
-  margin-right: -2px;
+.task-reading {
+  .step[data-chapter]{
 
-  .step {
-    .crumb-circle(2rem);
-    position: relative;
-    font-weight: 500;
-    cursor: pointer;
-    margin: 2px;
-    display: inline-block;
-    text-align: center;
-    background: @white;
-    color: @tutor-neutral-light;
-
-    &:hover,
-    &.active {
-      .tutor-icon-active();
-    }
+    background: @reading-color;
+    color: @tutor-white;
 
     &.active {
-      color: @tutor-primary;
+      color: @tutor-white;
     }
     &.completed {
-      color: @tutor-tertiary-light;
+      color: @tutor-white;
     }
-    &.status-correct {
-      background: @tutor-correct-background;
-      color: @tutor-correct-color;
+
+    &::before {
+      content: attr(data-chapter);
+      .crumb-text();
     }
-    &.status-incorrect {
-      background: @tutor-incorrect-background;
-      color: @tutor-incorrect-color;
+    i:not(.icon-incorrect):not(.icon-correct) {
+      display: none;
     }
   }
-}
\ No newline at end of file
+}
+
+.steps {
```

Comment:
Can these be scoped inside something like `.task-reading`? (Or enumerate all the possible containers)